docs/guides/migrating-from-cawemo.md

```diff
@@ -0,0 +1,56 @@
+---
+id: migrating-from-cawemo
+title: Migrating diagrams from Cawemo
+description: "Migrating diagrams from Cawemo to Camunda Platform 8 Web Modeler"
+---
+
+This guide steps through how to migrate BPMN diagrams created in Cawemo to Camunda Platform 8 Web Modeler.
+
+You should consider migrating diagrams from Cawemo to Camunda Platform 8 Web Modeler if:
+
+- You want to continue working on your diagrams on the latest Camunda Platform.
+- You are exploring automating your processes.
+- You are in need of certain features that can only be found in Camunda Platform 8 (e.g. [BPMN message buffering](https://docs.camunda.io/docs/components/concepts/messages/#message-buffering), big [multi-instance constructs](https://docs.camunda.io/docs/components/modeler/bpmn/multi-instance/), the new Connectors framework, or the improved collaboration features in web modeler).
```

Comment:
I would link to "the Connectors framework" (and remove "new" to keep this evergreen), and I would also link to the collaboration features you mention.